### PR TITLE
Non generic implementation on doubles

### PR DESCRIPTION
**Pull request by thomashilke**
Provide an implementation of the kinetic convolution which doesn't rely on the `IAlgebraicNumber` interface. This implementation may be useful for performances comparison, debugging, and contextes where robustness is not of utmost importance.

### CODE REVIEW

File: src/Hilke.KineticConvolution/Double/DirectionOrder.cs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+﻿namespace Hilke.KineticConvolution.Double
```
Comment (Gimly) [inline src/Hilke.KineticConvolution/Double/DirectionOrder.cs]: Reprendre celui déjà existant dans l'implémentation de base

File: src/Hilke.KineticConvolution/Double/Orientation.cs [hunk 2]
```diff
@@ -0,0 +1,8 @@
+namespace Hilke.KineticConvolution.Double
+{
+    public enum Orientation
```
Comment (Gimly) [inline src/Hilke.KineticConvolution/Double/Orientation.cs]: Pareil, reprendre l'implémentation de base

**Review by Gimly — APPROVED** [review]
Juste les deux points, le reste me semble OK

File: src/Hilke.KineticConvolution/Double/DirectionOrder.cs (hunk 1 above)
Comment (thomashilke) [reply src/Hilke.KineticConvolution/Double/DirectionOrder.cs]: Done

File: src/Hilke.KineticConvolution/Double/Orientation.cs (hunk 2 above)
Comment (thomashilke) [reply src/Hilke.KineticConvolution/Double/Orientation.cs]: Done

**Pull request merged**